Compile/interface/Producers/LeptonSFProducer.h

```diff
@@ -203,5 +87,6 @@ class LeptonRecoSFProducer : public LeptonSFProducer{
     std::string m_year;
 
   protected:
-    virtual float GetScaleFactor(float err_shift, KLV const& lepton) const;
+    virtual std::tuple<float, float> GetScaleFactorAndUnc(KLV const& lepton) const;
```

Comment:
You should also add an `overwrite` here, since this method is virtual in the parent class.